app/facades/food_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FoodFacade
+  def self.all_foods(food)
+    json_data = FoodService.foods(food)
+    food_hits = json_data[:totalHits]
```

Comment:
Nice work using the one API all to also get the count instead of making the API call a second time to accomplish this.